packages/tpetra/core/test/FECrsMatrix/FECrsMatrix_UnitTests.cpp

```diff
@@ -224,16 +224,16 @@ std::vector<std::vector<Scalar> > generate_fem1d_element_values() {
 }
 
 template<class Scalar, class Node>
-Kokkos::View<Scalar*[2], Kokkos::LayoutLeft, typename Node::device_type > generate_fem1d_element_values_kokkos() {
-  Kokkos::View<Scalar*[2], Kokkos::LayoutLeft, typename Node::device_type > mat;
+Kokkos::View<Scalar[2][2], Kokkos::LayoutLeft, typename Node::device_type > generate_fem1d_element_values_kokkos() {
+  Kokkos::View<Scalar[2][2], Kokkos::LayoutLeft, typename Node::device_type> mat ("fem1d_element_values");
```

Comment:
Consider this:
```c++
using Kokkos::view_alloc;
using Kokkos::WithoutInitializing;
using LL = Kokkos::LayoutLeft;
using DT = typename Node::device_type;

Kokkos::View<Scalar[2][2], LL, DT> mat
  (view_alloc ("fem1d_element_values", WithoutInitializing));
```

Reply:
This looks resolved.  Not sure why these little blocks need to be LayoutLeft, but there's nothing incorrect about that.

Reply:
We lost the `WithoutInitializing` bit -- how come?